tabs/src/components/app/views/RSCList.jsx

```diff
@@ -54,29 +59,35 @@ export function RSCList() {
                 else {
                     setAlert(true);
                 }
+                setIsLoading(false);
             }
         };
         getRSCList(context);
     }, [context, t]);
 
-
     return (
-        <div>
-            {RSCList?.length > 0 && (
-                <>
-                    <Table className="table" variables={{
-                        cellContentOverflow: 'none',
-                    }}
-                        header={{ items }}
-                        rows={RSCList}
-                        aria-label="RSC Table" />
-                </>
-            )}
-            {alert && (
-                <>
-                    <Alert danger content={t('RSC Descriptions.No resource available to grant permissions to')} />
-                </>
-            )}
-        </div>
+        <>
+        {isLoading ?
+            <Loader size='medium' label={t("RSC.loading")} />
+        :
+            <div>
+                {RSCList?.length > 0 && (
+                    <>
+                        <Table className="table" variables={{
+                            cellContentOverflow: 'none',
+                        }}
+                            header={{ items }}
+                            rows={RSCList}
+                            aria-label="RSC Table" />
```

Comment:
do we need to localize aria labels? so that screen reader and other accessibility tools know the localized content 👀